### DISCUSSION TITLE
Feature Request: 点击左上角时间戳时跳转到source，而不是显示卡片预览

### DISCUSSION THREAD

**Discussion by yett1873** [Quorafind/Obsidian-Thino#710] [Ideas 想法]
### Feature Requested | 请求的功能

如图，希望点击时间戳时能选择使用右侧列表中的更常用的其它功能（SOURCE、SHARE等），而不是显示一张可读的浮窗（READ）。个人日常完全用不到这个功能，好奇READ一般在什么场合使用？

### Relevant Screenshot | 相关截图

https://github.com/user-attachments/assets/40b971f0-c9bf-4ac0-bf40-d7fbdb961b68

![image](https://github.com/user-attachments/assets/f58feb9f-8cb4-4c2a-8509-da70ac3b15f5)

### Checklist | 检查清单

- [X] The feature would be useful to more users than just me. 该功能对于不仅仅是我自己的用户来说是有用的。

[a comment by an automated account is not shown]

**Comment by juestchaos** [comment]
1、对于很多长文记录的用户，比如一张卡片记录几千字长文的。
你这个问题，更像是一种讨论性质，所以移动到 discussion 区域

[a comment by an automated account is not shown]
